pkg/sql/sqlbase/structured.proto

```diff
@@ -1019,9 +1026,16 @@ message DatabaseDescriptor {
   // Needed for the descriptorProto interface.
   option (gogoproto.goproto_getters) = true;
 
+  // Shared descriptor fields. See the discussion at the top of TableDescriptor.
+
   optional string name = 1 [(gogoproto.nullable) = false];
   optional uint32 id = 2 [(gogoproto.nullable) = false,
       (gogoproto.customname) = "ID", (gogoproto.casttype) = "ID"];
+  // Last modification time of the descriptor.
+  optional util.hlc.Timestamp modification_time = 4 [(gogoproto.nullable) = false];
```

Comment:
I thought you were going to pull modtime, version and draining names into one bigger message for the new descriptors

Reply:
Discussed offline, copying the fields allows us to remove the migration. My preference evolved over time to preferring the messier protos and proto handling code in favor of the simpler go code given the existing structure.